CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/DefaultConversationDatetimeProvider.java

```diff
@@ -22,6 +22,7 @@
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
 import java.util.List;
+import java.util.Objects;
```

Comment:
this import isn't used according to sonar

Reply:
I agree